### DISCUSSION TITLE
[Question]: Frigate+ Image submission- labeling an object extending beyond the camera frame

### DISCUSSION THREAD

**Discussion by mstolpner** [blakeblackshear/frigate#9982] [Ask A Question] [answered]
### What is your question:

I have a stationary car which appears on all events from a single camera. This car does not completely fit in the camera frame. When I tag images with Frigate+, there is no was to build a label rectangle that would extend to outside of the image area. My question is - am I better off labeling a portion of this car in every submission or not labelling it at all?

**Comment by NickM-27** [comment, marked as answer]
you should label the car and just draw a box on all parts that are not outside the camera frame
